webrtc.html

```diff
@@ -11167,8 +11167,21 @@ <h2>Dictionary <a class="idlType">RTCStats</a> Members</h2>
               <code><a>RTCStats</a></code> objects, extracted from two
               different <code><a>RTCStatsReport</a></code> objects, MUST have
               the same id if they were produced by inspecting the same
-              underlying object. User agents are free to pick any format for
+              underlying object.</p>
+              <p>Stats ids MUST be alphanumerical. Stats ids MUST NOT be
```

Comment:
I added the "must be alphanumerical" too, which is separate issue but in the same spirit of not having people rely on particular implementations' way of generating ids; people might process and store stats dumps in databases, so they should know what format of ids they need to be able to handle.

Reply:
Makes me think we should say that the length of the IDs must not be longer than such and such...

Reply:
are hyphens allowed?
Better do allowed characters in a separate PR.